feat: add Azure/kubelogin

CrystalMethod opened the pull request:
[Azure/kubelogin](https://github.com/Azure/kubelogin): A Kubernetes credential (exec) plugin implementing azure authentication

```console
$ aqua g -i Azure/kubelogin
```

## How to confirm if this package works well

Reviewers aren't necessarily familiar with this package, so please describe how to confirm if this package works well.
Please confirm if this package works well yourself as much as possible.

Command and output

```console
$ kubelogin
login to azure active directory and populate kubeconfig with AAD tokens

Usage:
  kubelogin [flags]
  kubelogin [command]

Available Commands:
  completion         Generate the autocompletion script for the specified shell
  convert-kubeconfig convert kubeconfig to use exec auth module
  get-token          get AAD token
  help               Help about any command
  remove-tokens      Remove all cached tokens from filesystem

Flags:
  -h, --help          help for kubelogin
      --logtostderr   log to standard error instead of files (default true)
  -v, --v Level       number for the log level verbosity
      --version       version for kubelogin

Use "kubelogin [command] --help" for more information about a command.
```

CrystalMethod committed:
feat: add Azure/kubelogin

[Azure/kubelogin](https://github.com/Azure/kubelogin): A Kubernetes credential (exec) plugin implementing azure authentication

```console
$ aqua g -i Azure/kubelogin
```

## How to confirm if this package works well

Reviewers aren't necessarily familiar with this package, so please describe how to confirm if this package works well.
Please confirm if this package works well yourself as much as possible.

Command and output

```console
$
```

If files such as configuration file are needed, please share them.

```
```

Reference

-

suzuki-shunsuke commented:
You may be aware but let me share.
From v1.34.0, aqua has supported generating `version_overrides`.
https://aquaproj.github.io/docs/reference/scaffold-registry/#generate-version_overrides-by---deep-option

```console
$ aqua-registry --deep Azure/kubelogin
```

The result: https://github.com/aquaproj/generate-registry-deep/actions/runs/4309335805

suzuki-shunsuke commented:
Hmm. Unfortunately [the result](https://github.com/aquaproj/generate-registry-deep/actions/runs/4309335805) contains a bug, but it almost works well.

suzuki-shunsuke committed:
fix(Azure/kubelogin): scaffold config by --deep option and fix a bit

```console
$ aqua-registry --deep Azure/kubelogin
```

https://github.com/aquaproj/generate-registry-deep/actions/runs/4309335805

suzuki-shunsuke commented:
https://github.com/aquaproj/aqua-registry/actions/runs/4309419348/jobs/7516792195

```
ERRO[0001] install the package                           aqua_version=2.0.0-3 env=windows/arm64 error="check file_src is correct: check file_src is correct: get file_src: rename a file: rename C:\\Users\\runneradmin\\AppData\\Local\\aquaproj-aqua\\pkgs\\github_release\\github.com\\Azure\\kubelogin\\v0.0.27\\kubelogin-win-arm64.zip\\bin\\win_arm64\\kubelogin C:\\Users\\runneradmin\\AppData\\Local\\aquaproj-aqua\\pkgs\\github_release\\github.com\\Azure\\kubelogin\\v0.0.27\\kubelogin-win-arm64.zip\\bin\\win_arm64\\kubelogin.exe: The system cannot find the path specified." package_name=Azure/kubelogin package_version=v0.0.27 program=aqua registry=standard
```

```console
$ zipinfo kubelogin-win-arm64.zip 
Archive:  kubelogin-win-arm64.zip
Zip file size: 19769953 bytes, number of entries: 1
-rwxr-xr-x  3.0 unx 45436928 bx defN 23-Feb-23 07:31 bin/windows_arm64/kubelogin.exe
1 file, 45436928 bytes uncompressed, 19769741 bytes compressed:  56.5%
```

suzuki-shunsuke committed:
fix(Azure/kubelogin): support windows

suzuki-shunsuke commented:
Released. https://github.com/aquaproj/aqua-registry/releases/tag/v3.139.0
